pygmt/base_plotting.py

```diff
@@ -1178,6 +1194,8 @@ def text(
             Sets the pen used to draw a rectangle around the text string
             (see *clearance*) [Default is width = default, color = black,
             style = solid].
+        no_clip : bool
+            Do NOT clip text at map boundaries [Default will clip].
```

Comment:
```suggestion
            Do NOT clip text at map boundaries [Default is False, i.e. will clip].
```
Not so nice having a double negative here, so some extra text to clarify default `no_clip='False'` behaviour.